common/parser.hpp

```diff
@@ -14,6 +15,8 @@
 #endif
 
 #include "../third-party/rapidxml/rapidxml.hpp"
+#include "./utilities/string/string-utilities.h"
+using namespace utilities::strings;
```

Comment:
Same

Reply:
And note we're in a header -- should never even use `using` in a header, either.

Reply:
Also:
`parser.hpp` is not in the CMakeLists.txt list of files. Can you add it, please?

Reply:
done